stats/record.go

```diff
@@ -31,10 +31,17 @@ func init() {
 	}
 }
 
+// Recorder is a subset of the view.Meter interface which only includes
+// the Record method, to avoid circular imports between stats and view.
```

Comment:
keep this definition and simply include this in the Meter.

Reply:
Done. I updated the comment, too.

What do you think of changing the second argument to Record to be `[]stats.Measurement` rather than the current `interface{}`?

Reply:
Thanks, done.

Reply:
> Done. I updated the comment, too.
> 
> What do you think of changing the second argument to Record to be `[]stats.Measurement` rather than the current `interface{}`?



> Done. I updated the comment, too.
> 
> What do you think of changing the second argument to Record to be `[]stats.Measurement` rather than the current `interface{}`?

that is better.

Reply:
It turns out that this introduces an import cycle:

    import cycle not allowed
     package go.opencensus.io/stats/view
    	imports go.opencensus.io/stats
    	imports go.opencensus.io/stats/internal
    	imports go.opencensus.io/stats

Would you prefer that I move internal.DefaultRecorder out of `internal`, or do something like create an InternalMeasurement struct or interface and embed it in stats.Measurement?

Reply:
I would say leave it as interface.

Reply:
It turns out that `stats.Measurement.Measure()` returns a stats.Measure, which seemed like a bridge too far in terms of putting things into Internal.

Take a look at 6003f86 for what that change would look like. I'm slightly concerned about moving DefaultRecorder from the internal package (and making it more public/obvious).

What do you think?

Reply:
Reverted this to stick to interfaces.